fixtures/simple-example.format

```diff
@@ -85,6 +85,9 @@ test-suite doctests
     , simple-example
     , template-haskell
 
-  ghc-options:      -Wall -threaded
+  ghc-options:
+    -Wall
+    -threaded
```

Comment:
I don't like how this reformats here. If it fits on one line, it should be there.

Reply:
I disagree with fitting ghc-options on one line where possible. I prefer to always have them on separate lines because that's the easiest way for me to diff between two sets of ghc-options and see what has changed. It also generates the easiest-to-read diff for my code reviewers.

That said, I totally get if you don't want the ghc-options functionality to work this way - I'm more than happy to close this PR and just maintain a fork that adds this patch.

Reply:
Feel free to fork.